trie/syncCreator.go

```diff
@@ -1,17 +1,22 @@
 package trie
 
 import (
+	"context"
 	"fmt"
-
-	"github.com/ElrondNetwork/elrond-go/state/temporary"
 )
 
 const initialVersion = 1
 const secondVersion = 2
 
+// TrieSyncer synchronizes the trie, asking on the network for the missing nodes
```

Comment:
group above consts in `const ( .... ) ` block

Reply:
done